build_docs.py

```diff
@@ -745,6 +745,9 @@ def build(self):
         blurb = self.venv / "bin" / "blurb"
 
         if self.includes_html:
+            # For opengraph previews (requires matplotlib)
+            sphinxopts.append("-t create-social-cards")
```

Comment:
Or to make clear this is a tag?
```suggestion
            sphinxopts.append("--tag create-social-cards")
```

Reply:
I think this would fail when rebuilding Python 3.11 and earlier, as they use versions of Sphinx that are too old (per the README).